packages/trader/src/AppV2/Utils/trade-types-utils.tsx

```diff
@@ -47,3 +47,26 @@ export const getTradeTypesList = (contract_types_list: ReturnType<typeof useTrad
                 ![TRADE_TYPES.VANILLA.PUT, TRADE_TYPES.TURBOS.SHORT, TRADE_TYPES.RISE_FALL_EQUAL].includes(value)
         );
 };
+
+export const focusAndOpenKeyboard = (element: HTMLInputElement | null) => {
```

Comment:
IOS prevent input from focusing with showing keyboard before user clicks on it. The only solution that helped me was taken from [here](https://stackoverflow.com/questions/54424729/ios-show-keyboard-on-input-focus).
Before, I tried with `autofocus` attribute, also called `focus()` and `click()` as it is and inside of timer. Nothing helps and autofocus is even not supported in IOS.
<img width="1361" alt="Screenshot 2024-07-29 at 6 44 50 PM" src="https://github.com/user-attachments/assets/dad6d62e-5409-484b-9bfe-ce96686d73cb">

Reply:
FYI: If I just do 
```
return setTimeout(() => {
            element.focus();
            element.click();
        }, 300);
```
without creating element, putting focus on it and removing the element - it won't work.
If you now less hacker way, please share.